grid/recruitment/get-stage-changes/profile.supr

```diff
@@ -0,0 +1,198 @@
+"""
+Retrieve Stage Changes
+Lists stage changes for a candidate.
+"""
+name = "recruitment/get-stage-changes"
+version = "1.0.0"
+
+"""
+Get Stage Changes
+Retrieves a list of stages for specific candidate throughout his recruitment process. The candidate does not have to be applied for a job.
```

Comment:
```suggestion
Retrieves a list of stages for specific candidate or lead throughout the recruitment process
```
We have `CreateCandidate` or `CreateLead` use cases. We should use the `Lead` term here to be consistent with the use cases.